components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties

```diff
@@ -1,3 +1,21 @@
+#
+# Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
```

Comment:
No, actualy speaking above file is not created in 2014. This file is available in 4.0.0 branch as well (which means in  2013).. It seems, you have added a license header for these files. IMO, If you are not aware about the exact year that file created. Then better to add current year when u added license header..